tasks/source.yml

```diff
@@ -0,0 +1,62 @@
+---
+# TODO: Replace this with `manage.py version` in the next release
+- name: Check if the current PostMaster version is {{ postmaster_version }}
+  command: "grep -Fxq \"__version__ = '{{ postmaster_version }}'\" /opt/postmaster/git/postmaster/__init__.py"
```

Comment:
This is so that if you are performing an upgrade, it checks the currently installed version. On a fresh install, this errors silently and in the subsequent task `postmaster_desired_version` gets set to `False`.